### ISSUE TITLE
Transmission - merge complete and incomplete selections

### ISSUE THREAD

**Issue by VladFlorinIlie** [truenas/apps#748]
Hello,

Due to the way file moves from incomplete directory to complete are implemented in transmission, creating two volume binds is bad for performance (it freezes the whole app for minutes). Basically, because the binds are seen as two file systems, instead of a simple move, the app does a copy + remove.

The solution would be choosing in the app template only one higher directory to be mounted in the container that will then contain two directories for complete and incomplete torrents.

**Comment by stavros-k** [comment]
Hello, 

I think you can uncheck the `Enable Incomplete Dir` and transmission will use the same path for both.

**Comment by VladFlorinIlie** [comment]
That would disable the incomplete directory entirely. I would still like to keep complete and incomplete files separately.

**Comment by stavros-k** [comment]
Where are the incomplete files/chunks are stored when incomplete directory is disabled?

Also, can you verify that the host paths you are using for downloads and incomplete are NOT different datasets?

**Comment by VladFlorinIlie** [comment]
Without the incomplete directory, the chunks are stored in the "normal" download directory (basically their final location).

On the host, both directories are part of the same dataset. But the transmission container sees them as two filesystems (with two different entries returned by the mount command).

I have tested with a custom compose yaml where I copied the rendered yaml from transmission and used only one upper level directory (so one volume for both complete and incomplete) with two sub directories and the problem does not appear anymore.
